### PR TITLE
Added Get-RsRestItemAccess and test.

### PR DESCRIPTION
**Pull request by SQLvariant**
Added a new function Get-RsRestItemAccess to be the REST/PS7 equivalent of Get-RsCatalogItemRole.

Changes proposed in this pull request:
 - Added a new function Get-RsRestItemAccess.ps1
 - Added a new pester test Get-RsRestItemAccess.Tests.ps1

How to test this code:
 - Run the pester test.

Has been tested on (remove any that don't apply):
 - PowerShell 7.1 and above
 - Windows 10
 - PBIRS

### CODE REVIEW

File: ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1 [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                    }
+    
+    
+                }
+            }            
+            return $catalogItemRoles
+        }
+        catch
+        {
+            throw (New-Object System.Exception("Failed to get cache refresh plan for '$RsItem': $($_.Exception.Message)", $_.Exception))
```
Comment (jtarquino) [inline ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1:185]: please update the error message

File: ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1 [hunk 2]
```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2020 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsRestItemAccess
+{
+    <#
+        .SYNOPSIS
+            This script retrieves access to SQL Server Reporting Services Instance or Power BI Report Server Instance from users/groups.
```
Comment (jtarquino) [inline ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1:8]: I would mention access policies

File: ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1 (hunk 1 above)
Comment (SQLvariant) [reply ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1]: Done.

File: ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1 (hunk 2 above)
Comment (SQLvariant) [reply ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccess.ps1]: Done.

File: ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccessPolicy.ps1 [hunk 3]
```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2020 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsRestItemAccessPolicy
+{
+    <#
+        .SYNOPSIS
+            This script retrieves access polocies to SQL Server Reporting Services Instance or Power BI Report Server Instance from users/groups.
```
Comment (jtarquino) [inline ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccessPolicy.ps1:8]: typo policies

File: ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccessPolicy.ps1 [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                    }
+    
+    
+                }
+            }            
+            return $catalogItemRoles
+        }
+        catch
+        {
+            throw (New-Object System.Exception("Failed to get access polocies for '$RsItem': $($_.Exception.Message)", $_.Exception))
```
Comment (jtarquino) [inline ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccessPolicy.ps1:185]: typo policies

**Pull request merged**